Cleaning up support for MacOS & Homebrew (#1799)
advancedtelematic/aktualizr: docs/ota-client-guide/modules/ROOT/pages/generate-selfsigned-root.adoc

  "${fleet_name}/fleet_root_ca.cnf" -out "${fleet_name}/fleet_root_ca.crt"
----
+

> WARNING: These commands are designed for openssl 1.1 or higher. If you are using an older version, or if you are using LibreSSL, it may not work.

Reviewer: ```suggestion
WARNING: This command string is designed for OpenSSL 1.1 or higher. If you use an older version or LibreSSL, it may not work.
```
